dev/com.ibm.ws.security.kerberos.java8/.project

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>com.ibm.ws.security.kerberos.java8</name>
+	<name>com.ibm.ws.security.kerberos.s4u</name>
```

Comment:
But why?

Reply:
This is bundle is for s4u feature and it's required JDK 1.8 and higher. I think it's better bundle name unless you think we may hit some problems that I over look.